app/models/node.rb

```diff
@@ -920,8 +924,14 @@ def self.research_notes
                .group('node.nid')
                .collect(&:nid)
 
-    Node.where(type: 'note')
-                .where('node.nid NOT IN (?)', nids)
+    notes = Node.where(type: 'note')
+
+    # if the nids are empty, it should not be used in the query
```

Comment:
Can you also write why we donot have to use `nids` in comment?

Reply:
sure!

Reply:
Done!